src/Microsoft.IdentityModel.Logging/IdentityModelTelemetryUtil.cs

```diff
@@ -130,6 +130,12 @@ internal static void SetTelemetryData(HttpRequestMessage request)
                 request.Headers.Remove(parameter.Key);
                 request.Headers.Add(parameter.Key, parameter.Value);
             }
+
+            if (additionalHeaders != null)
+            {
+                foreach (var parameter in additionalHeaders)
```

Comment:
nit: would it make more sense to call this variable 'header' rather than 'parameter'?

Reply:
@RojaEnnam can you make this change and then we can approve.

Reply:
Done.